Problem when building

Hi all! 

When I build the project I get this:

`#0 28.22 /tmp/hipSYCL-installer/src/compiler/cbs/IRUtils.cpp: In function 'bool hipsycl::compiler::utils::checkedInlineFunction(llvm::CallBase*, llvm::StringRef)':
#0 28.22 /tmp/hipSYCL-installer/src/compiler/cbs/IRUtils.cpp:138:59: error: converting to 'bool' from 'std::nullptr_t' requires direct-initialization [-fpermissive]
#0 28.22   138 |   llvm::InlineResult ILR = llvm::InlineFunction(*CI, IFI, nullptr);
#0 28.22       |                                                           ^~~~~~~
#0 28.22 In file included from /tmp/hipSYCL-installer/src/compiler/cbs/IRUtils.cpp:41:
#0 28.22 /opt/hipSYCL/llvm/include/llvm/Transforms/Utils/Cloning.h:267:34: note:   initializing argument 3 of 'llvm::InlineResult llvm::InlineFunction(llvm::CallBase&, llvm::InlineFunctionInfo&, bool, llvm::AAResults*, bool, llvm::Function*)'
#0 28.22   267 |                             bool MergeAttributes = false,
`

Does someone know how to solve it? Thank you very much!

--- a/src/compiler/cbs/IRUtils.cpp
+++ b/src/compiler/cbs/IRUtils.cpp
@@ -135,7 +135,7 @@ bool checkedInlineFunction(llvm::CallBase *CI, llvm::StringRef PassPrefix) {
     HIPSYCL_DEBUG_WARNING << PassPrefix << " failed to inline function <" << calleeName << ">: '"
                           << ILR.message << "'\n";
 #else
-  llvm::InlineResult ILR = llvm::InlineFunction(*CI, IFI, nullptr);
+  llvm::InlineResult ILR = llvm::InlineFunction(*CI, IFI);
   if (!ILR.isSuccess()) {
     HIPSYCL_DEBUG_WARNING << PassPrefix << " failed to inline function <" << CalleeName << ">: '"
                           << ILR.getFailureReason() << "'\n";

